utils/src/lib.rs

```diff
@@ -18,8 +21,73 @@ use std::{
     fmt,
 };
 
+use std::{fmt, path::PathBuf, str::FromStr};
+
 use tokio::runtime::{Handle, Runtime};
 
+#[derive(Debug)]
+pub struct CastAccount {
+    pub address: Address,
+    pub path: Option<DerivationPath>,
+    pub kind: CastAccountType,
+}
+
+#[derive(Debug)]
+pub enum CastAccountType {
```

Comment:
This feels like its a bit redundant, I've seen similar types in ethers elsewehere